tests/core/test_fileio.py

```diff
@@ -120,8 +121,10 @@ def test_save_restore_from_nemo_file(self, asr_model):
         with tempfile.NamedTemporaryFile() as fp:
             filename = fp.name
 
-            # Save model.
-            asr_model.save_to(save_path=filename)
+            # Save model (with random artifact).
+            with tempfile.NamedTemporaryFile() as artifact:
+                asr_model.register_artifact(conf_path=None, src=artifact.name)
```

Comment:
What happens when config path is None? I see self.artifacts just appends (None, path)?

Reply:
just ignores it, still adds file.